pkg/rtc/supervisor/participant_supervisor.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	monitorInterval = 3 * time.Second
+	monitorInterval = 1 * time.Second
```

Comment:
initially reduced it because I thought the events will be fired with a delay. It turns out they were handled in the `update` function. @boks1971 let me know if this should be reverted.

Reply:
I think it is fine to leave it at this. Wanted to keep it low overhead. So used 3 seconds. This is lightweight, 1 second is fine.